docs/auth0_api.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+---
+## auth0 api
+
+Makes an authenticated HTTP request to the Auth0 Management API
+
+### Synopsis
+
+Makes an authenticated HTTP request to the Auth0 Management API and prints the response as JSON.
+
+The method argument is optional, and when you don’t specify it, the command defaults to GET for requests without data and POST for requests with data.
+
+Auth0 Management API Docs:
+  https://auth0.com/docs/api/management/v2
+
+Available Methods:
+  GET, POST, PUT, PATCH, DELETE
+
+```
+auth0 api <method> <uri> [flags]
+```
+
+### Examples
+
+```
+auth0 api "/stats/daily?from=20221101&to=20221118"
+auth0 api get "/tenants/settings"
+auth0 api clients --data "{\"name\":\"ssoTest\",\"app_type\":\"sso_integration\"}"
+
+```
+
+### Options
+
+```
+  -d, --data string   JSON data payload to send with the request.
+  -h, --help          help for api
+```
+
+### Options inherited from parent commands
+
+```
+      --debug           Enable debug mode.
```

Comment:
How could we leverage this flag to be useful, considering that this command requires more manual construction/input from the user, and thus there's a bigger surface area for things to go wrong?

Reply:
🤔 I'll come back with an update.

Reply:
Updated in: [`2e5ac1e` (#531)](https://github.com/auth0/auth0-cli/pull/531/commits/2e5ac1e6cead6a25afe89f7c69b0dfe596584054).